docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
+  database:
+    image: postgres:17.2
+
+    ports:
+      - 5432:5432
```

Is it worth aliasing the host port to avoid it failing to bind for anyone who has PSQL natively installed
```suggestion
      - 54320:5432
```

---

sure!